rotkehlchen/chain/evm/decoding/structures.py

```diff
@@ -53,21 +53,31 @@ class EnricherContext(DecoderBasicContext):
 class DecodingOutput(NamedTuple):
     """
     Output of decoding functions
-    matched_counterparty is optionally set if needed for decoder rules that matched
+
+    - event can be returned if the decodeing method has generated a new event and it needs to be
+    added to the list of other decoded events.
+    - action_items is a list of actions to be performed later automatically or to be passed
+    in further decoding methods.
+    - matched_counterparty is optionally set if needed for decoder rules that matched
     and is used in post-decoding rules like in the case of balancer
     """
     event: Optional['EvmEvent'] = None
     action_items: list[ActionItem] = []
     matched_counterparty: Optional[str] = None
+    refresh_balances: bool = False
```

Comment:
You did not add the explanation for refresh balances in this docstring. Only on the next one. Please don't make me have to spell out every needed change.